parm/atm/berror/staticb_gsibec.yaml

```diff
@@ -0,0 +1,39 @@
+background error:
+  covariance model: hybrid
+  components:
+  - covariance:
+      covariance model: SABER
+      saber blocks:
+      - saber block name: gsi covariance
+        saber central block: true
+        input variables: &control_vars [eastward_wind,northward_wind,air_temperature,surface_pressure,
+                                 specific_humidity,cloud_liquid_ice,cloud_liquid_water,
+                                 mole_fraction_of_ozone_in_air]
+        output variables: *control_vars
+        gsi akbk: &akbk $(fv3jedi_fix_dir)/akbk.nc4
+        gsi error covariance file: &gsiberr $(fv3jedi_staticb_dir)/gsi-coeffs-gfs-global-l127x192y96.nc4
```

Comment:
is there a way we can make the resolution here a variable? It might require changes to the YAML generation scripts to do so.

Reply:
@RussTreadon-NOAA I think that is the solution that makes the most sense to me. It mimics what we do for GSI now (`./berror_stats` comes from a number of different files). I'm already doing something similar with the `field metadata override` entry in YAML, keeping it the same name but copying different YAML files depending on if it is GFS or GFS-aerosol.

Reply:
@CoryMartin-NOAA, let me test this using my local copy of `gdas_fix_root`.

Reply:
@CoryMartin-NOAA , renaming gsibec files in the fix directory along with the corresponding change in `staticb_gsibec.yaml` works.   In going through this exercise and conversations with Ricardo, `staticb_gsibec.yaml` has been simplified.  

Ricardo pointed out that `fv3jedi_var.x` can process both binary global_berror files directly from GSI fix as well as the netcdf counterparts.   fv3-jed executable `write_berror_global.x` converts GSI global_berror files to netcdf.  It was found that some of the conversions were done with the wrong number of latitudes.  This has been corrected.  Using the binary or netcdf global_berror files does not alter `fv3jedi_var.x` results.    

We need to repopulate 
- Hera:  `/scratch1/NCEPDEV/da/Cory.R.Martin/GDASApp/fix/gsibec`
- Orion;  `/work2/noaa/da/cmartin/GDASApp/fix/gsibec`

with files from `/scratch1/NCEPDEV/da/Russ.Treadon/GDASApp/fix/gsibec`   Each case directory now contains three files
1. `gfs_gsi_global.nml` - gsibec namelist specifying number of latitudes and longitudes in the berror file, other sections of the namelist are currently identical between the various cases
2. `global_berror.f77` - binary global_berror file from GSI fix
3. `gsi-coeffs-gfs-global.nc4` - netcdf counterpart to binary global_berror file

It is not necessary to retain both the binary and netcdf formats long term.   Having both for the time being allows me to run `gsi.x` and `fv3jedi_var.x` with exactly the same global_berror file.

@KateFriedman-NOAA , we need to repopulate glopara `fix/gdas/gsibec/20221031`.   Please remove the existing case directories and copy the case directories from `/scratch1/NCEPDEV/da/Russ.Treadon/GDASApp/fix/gsibec`.

Reply:
@RussTreadon-NOAA Question...I see a new case folder in the updated copy called `C766`. Should that be `C768`? The other folders correspond to regular resolution values, so wanted to check.

Reply:
@RussTreadon-NOAA my directories have been updated. @KateFriedman-NOAA I had the same thought and assumed it was a typo and changed mine to be `C768`

Reply:
You are right @CoryMartin-NOAA and @KateFriedman-NOAA . Directory `C766` should be `C768`. I moved `C766` to `C768` in Hera `/scratch1/NCEPDEV/da/Russ.Treadon/GDASApp/fix/gsibec`.

Reply:
The updated gsibec files have been copied into the fix set on Hera:
```
[glopara@hfe10 20221031]$ pwd
/scratch1/NCEPDEV/global/glopara/fix/gdas/gsibec/20221031
[glopara@hfe10 20221031]$ rsync -azv /scratch1/NCEPDEV/da/Russ.Treadon/GDASApp/fix/gsibec/C* .
sending incremental file list
C192/
C192/gfs_gsi_global.nml
C192/global_berror.f77
C192/gsi-coeffs-gfs-global.nc4
C384/
C384/global_berror.f77
C384/gsi-coeffs-gfs-global.nc4
C768/
C768/gfs_gsi_global.nml
C768/global_berror.f77
C768/gsi-coeffs-gfs-global.nc4
C96/
C96/gfs_gsi_global.nml
C96/global_berror.f77
C96/gsi-coeffs-gfs-global.nc4

sent 204,453,688 bytes  received 256 bytes  16,356,315.52 bytes/sec
total size is 251,807,096  speedup is 1.23
```

Rsyncing them to the other platforms now.

Reply:
@KateFriedman-NOAA , thank you for updating `fix/gdas/gsibec` on Hera. Orion, and WCOSS2 (the only machines I checked).   

We need to remove the `gsi-coeffs-gfs-global-l*x*y*.nc4` files in C192, C384, and C96.   These files are obsolete.   If you want, I can log into the global role account on Hera and WCOSS2 and remove the obsolete files.   Just let me know.  

You own the Orion fix files.

Reply:
Removed from the Hera set:
```
[glopara@hfe10 20221031]$ rm */gsi-coeffs-gfs-global-l*x*y*.nc4
rm: remove regular file 'C192/gsi-coeffs-gfs-global-l127x384y192.nc4'? y
rm: remove regular file 'C384/gsi-coeffs-gfs-global-l127x768y386.nc4'? y
rm: remove regular file 'C96/gsi-coeffs-gfs-global-l127x192y96.nc4'? y
[glopara@hfe10 20221031]$ ll *
C192:
total 30828
-rw-r--r-- 1 glopara da     3593 Nov  2 17:47 gfs_gsi_global.nml
-rw-r--r-- 1 glopara da 15183556 Nov  2 19:32 global_berror.f77
-rw-r--r-- 1 glopara da 16375984 Nov  2 19:32 gsi-coeffs-gfs-global.nc4

C384:
total 63640
-rw-r--r-- 1 glopara da     3595 Oct 28 15:08 gfs_gsi_global.nml
-rw-r--r-- 1 glopara da 31396036 Nov  2 19:30 global_berror.f77
-rw-r--r-- 1 glopara da 33761200 Nov  2 19:29 gsi-coeffs-gfs-global.nc4

C768:
total 136172
-rw-r--r-- 1 glopara da     3596 Nov  2 19:27 gfs_gsi_global.nml
-rw-r--r-- 1 glopara da 67359940 Nov  2 19:28 global_berror.f77
-rw-r--r-- 1 glopara da 72070576 Nov  2 19:26 gsi-coeffs-gfs-global.nc4

C96:
total 15284
-rw-r--r-- 1 glopara da    3592 Nov  2 16:41 gfs_gsi_global.nml
-rw-r--r-- 1 glopara da 7519684 Nov  2 19:34 global_berror.f77
-rw-r--r-- 1 glopara da 8125744 Nov  2 19:34 gsi-coeffs-gfs-global.nc4
```

Removing from the other sets now too (WCOSS2, Orion, Jet).

Reply:
Thank you, @KateFriedman-NOAA

Reply:
Now removed from the WCOSS2, Orion, and Jet sets.